.github/scripts/prepare-new-pr.sh

```diff
@@ -34,6 +34,9 @@ if [ $COUNT -eq 1 ]; then
     CHANGE_TYPE=$(awk -F': ' '/^change_type:/ {print $2}' "$PR_CHANGELOG_PATH/$CHLOG" | xargs)
     echo $CHANGE_TYPE
     gh pr edit "${PR}" --add-label "${CHANGE_TYPE}" || true
```

Comment:
I believe you can add [multiple labels at once](https://cli.github.com/manual/gh_pr_edit) at once with something like `gh pr edit "${PR}" --add-label "${CHANGE_TYPE},{$AREA}"`

Reply:
Done